chaosk8s/pod/actions.py

```diff
@@ -17,19 +15,27 @@
 
 def terminate_pods(label_selector: str = None, name_pattern: str = None,
                    all: bool = False, rand: bool = False,
-                   ns: str = "default", secrets: Secrets = None):
+                   ns: str = "default", secrets: Secrets = None,
+                   mode: str = "fixed", qty: int = 1):
```

Comment:
The toolkit passes data as keywords not positional, so I think it's safe to assume we can insert new parameters before other existing ones.

Please, keep `secrets` as the last parameter in this signature.